spring-boot/spring-boot-openapi-impl/specification/pom.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.reflectoring</groupId>
+    <artifactId>specification</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <java.version>11</java.version>
+
+        <swagger-annotations-version>1.5.22</swagger-annotations-version>
```

Comment:
Hi @thombergs, yes. They are necessary, otherwise the generated classes are missing some imports and they doesn't work at all. 

I found it also strange.